### PR TITLE
PR for release v2023.0.4

### PR DESCRIPTION
**Pull request by adiganga2002**
opendsu-related:
			EPI-210 EPI 129 - Recovery Passphrases are Encrypted on disk: https://pharmaledger.atlassian.net/jira/software/c/projects/EPI/boards/2?modal=detail&selectedIssue=EPI-210
			EPI-208 Add log for EPI 127 - DSU adheres to PLA Cryptographic Standard: https://pharmaledger.atlassian.net/jira/software/c/projects/EPI/boards/2?modal=detail&selectedIssue=EPI-208
			EPI-211 EPI 108 - Key Rotation - Log: https://pharmaledger.atlassian.net/jira/software/c/projects/EPI/boards/2?modal=detail&selectedIssue=EPI-211
			EPI-266 - Cloud-agent cause by the instability of reading embedded files: https://pharmaledger.atlassian.net/jira/software/c/projects/EPI/boards/2?modal=detail&selectedIssue=EPI-266
			EPI-191 - Max call stack size exceeded crash: https://pharmaledger.atlassian.net/jira/software/c/projects/EPI/boards/2?modal=detail&selectedIssue=EPI-191
https://pharmaledger.atlassian.net/browse/EPI-310?atlOrigin=eyJpIjoiNDE1Y2EzZGYxMGJiNDYzZDg5MmVjNjY4NDQwMjQyNTIiLCJwIjoiaiJ9
https://pharmaledger.atlassian.net/browse/EPI-311?atlOrigin=eyJpIjoiZGExYTlhODBiZDVmNDlkN2ExNzk4MTE2Y2MwOWE4NTUiLCJwIjoiaiJ9
https://pharmaledger.atlassian.net/browse/EPI-319?atlOrigin=eyJpIjoiNTM0NWIwNDQwMDAxNDkzNzg0NWIxNzc3YTIyMjI5NjUiLCJwIjoiaiJ9
https://pharmaledger.atlassian.net/browse/EPI-319?atlOrigin=eyJpIjoiNTM0NWIwNDQwMDAxNDkzNzg0NWIxNzc3YTIyMjI5NjUiLCJwIjoiaiJ9
https://pharmaledger.atlassian.net/browse/EPI-336?atlOrigin=eyJpIjoiMWE1ZWZjY2I3MzcyNDIzNmFhOGFlY2YwMjIxZWZjODgiLCJwIjoiaiJ9

octopus:
			EPI-243 - Error handling in octopus/scripts/setEnv: https://pharmaledger.atlassian.net/jira/software/c/projects/EPI/boards/2?modal=detail&selectedIssue=EPI-243

lpwa:
			EPI-124 - Privacy Policy & TC must be available in local languages : https://pharmaledger.atlassian.net/jira/software/c/projects/EPI/boards/2?modal=detail&selectedIssue=EPI-124
			EPI-188 - Fixing the performance issue with gtinOwner and leaflets component :https://pharmaledger.atlassian.net/jira/software/c/projects/EPI/boards/2?modal=detail&selectedIssue=EPI-188
			EPI-192 - Ensure that /leaflets api calls include the headers from bdns config: https://pharmaledger.atlassian.net/jira/software/c/projects/EPI/boards/2?modal=detail&selectedIssue=EPI-192
			EPI-194 - in PWA general settings, the app version shows: 2023.0.1 instead of 2023.0.2: https://pharmaledger.atlassian.net/jira/software/c/projects/EPI/boards/2?modal=detail&selectedIssue=EPI-194
			EPI-195 - Information/error message in T&C too: https://pharmaledger.atlassian.net/jira/software/c/projects/EPI/boards/2?modal=detail&selectedIssue=EPI-195
			EPI-199 - Font size of the LWA and leaflet is not adjustable by the browser settings: https://pharmaledger.atlassian.net/jira/software/c/projects/EPI/boards/2?modal=detail&selectedIssue=EPI-199
			EPI-202 - New message error in LPWA when the remote system is busy: https://pharmaledger.atlassian.net/jira/software/c/projects/EPI/boards/2?modal=detail&selectedIssue=EPI-202
			EPI-253 - Not full error description is shown on screen in LWA: https://pharmaledger.atlassian.net/jira/software/c/projects/EPI/boards/2?modal=detail&selectedIssue=EPI-253
https://pharmaledger.atlassian.net/browse/EPI-363?atlOrigin=eyJpIjoiMmU2NWRjZTU1Y2IzNGU1NGEyYTAzMWMyMTlkZjI2NGMiLCJwIjoiaiJ9
https://pharmaledger.atlassian.net/browse/EPI-325?atlOrigin=eyJpIjoiZmYzZGVmNDlmNGUwNDAzYTlmMGFlNzI0MjNjMTM2MjMiLCJwIjoiaiJ9
https://pharmaledger.atlassian.net/browse/EPI-327?atlOrigin=eyJpIjoiZDBhYzc1MmJkNWExNDM1OGI1Y2U3OTA1MDY1OTNjNTciLCJwIjoiaiJ9

### CODE REVIEW

File: release-notes.txt [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#reslese v2023.0.3
```
Comment (PL-James) [inline release-notes.txt:1]: Incorrect release version specified. Please Update

File: apihub-root/lightweight-pwa/environment.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export default {
   "epiDomain": "localhost.epi",
   "epiProtocolVersion": 1,
-  "appBuildVersion": "git 2",
+  "appBuildVersion": "2023.0.2.a",
```
Comment (PL-James) [inline apihub-root/lightweight-pwa/environment.js:4]: Query: Should this be matching the release version?

**Review by PL-James — CHANGES_REQUESTED** [review]
Queries on version numbers outstanding.

**Comment by salboaie** [comment]
@PL-James we did those changes on wrong versions

**Review by PL-Marco — APPROVED** [review]

**Review by PL-James — APPROVED** [review]

**Comment by PL-James** [comment]
Blocks resolved. PR approved by QA.

**Comment by PL-Marco** [comment]
Great step towards production. 

This is a test run for the final release 2023.1.0

**Pull request merged**